packages/arb-bridge-eth/contracts/vm/ArbFactory.sol

```diff
@@ -44,7 +44,7 @@ contract ArbFactory is CloneFactory {
         uint128 _gracePeriodTicks,
         uint128 _arbGasSpeedLimitPerTick,
         uint64 _maxExecutionSteps,
-        uint64[2] calldata _maxTimeBoundsWidth,
+        uint64 _maxBlockBoundsWidth,
```

Comment:
Why do we still need `_maxBlockBoundsWidth`?

Reply:
Timebounds have now been completely removed